static/app/views/replays/replayTable/index.tsx

```diff
@@ -166,6 +168,7 @@ function ReplayTable({
                       referrer={referrer}
                       showUrl={false}
                       eventView={eventView}
+                      referrer_table="errors-table"
```

Comment:
I'm seeing the field hard-coded into the `ReplayCell` component and like, ugh, is the name `referrer_table` correct?

I think it's fine, because we're using the cell name as a proxy for the whole table. The cells are mutually exclusive, `visibleColumns` should not include both `ReplayColumn.MOST_ERRONEOUS_REPLAYS` as well as `ReplayColumn.REPLAY`.